api/src/main/java/org/openmrs/hl7/impl/HL7ServiceImpl.java

```diff
@@ -393,6 +393,18 @@ public HL7InError getHL7InErrorByUuid(String uuid) throws APIException {
 		return dao.getHL7InErrorByUuid(uuid);
 	}
 	
+	/** 
+ 	 * @param idNum id number
+ 	 * @param fName family name
+ 	 * @param gName given name
+ 	 * @return error string. User can not be resolveUserId
+ 	 */
+ 	public String errorFindingUser(String idNum, String fName, String gName) {
+ 		String cantFindUser = "Error resolving user with id '" + idNum + "' family name '" + fName
+ 				  + "' and given name '" + gName + "'";
+ 		return cantFindUser;
+ 	}
```

Comment:
Did you intend for this method to be part of the API? If you are just wrapping redundant code, then this method should be private as it is not part of the api.